Microsoft.Toolkit.Uwp.UI/Extensions/NavigationView/NavigationViewExtensions.cs

```diff
@@ -0,0 +1,181 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.Toolkit.Uwp.UI.Extensions
+{
+    /// <summary>
+    /// Set of extensions for the <see cref="NavigationView"/> control.
+    /// </summary>
+    [Bindable]
+    public class NavigationViewExtensions
```

Comment:
Should this be static?

Reply:
Yup, missed it, testing and will update.  Thanks!

Reply:
Nope, it needs to be non-static.

Reply:
Interesting, why? The other attached properties are in static classes

Reply:
I tried, but then the XAML parser says it can't find the attached property.

I know our extension method classes are static, but the other UI ones aren't (like TextBoxMask/RegEx).  Though I think the WebView one was which seemed odd, something to check during the bug bash.

Reply:
I think I ran into that issue when I was creating the composition animations - I had to use them in the actual sample page (in addition to the .bind file) so the XAML parser can find them.

Reply:
Considering I built the system, you think I'd remember to check for that.  Let me test that out.

I was digging around in the docs, and there may be an event I can use to help load them at runtime, but I haven't looked into it yet.

Reply:
Right, I was going to include it, but it wouldn't let me include the NavigationView in a 14393 app, and I couldn't use the conditional XAML as we're not a 15063 min app.

Reply:
As discussed, we'll leave it for now and fix it in 3.0.